.travis.yml

```diff
@@ -39,57 +41,86 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test -V' OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
 
         # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time
-        # OPTIONAL_DEPS needed because the plot_directive in sphinx needs them
+        # runs for a long time. All dependencies are needed because the
+        # plot_directive in sphinx needs them
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='pyephem pytest-mpl wcsaxes astroquery'
         - os: linux
-          env: PYTHON_VERSION=3.4 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=3.5 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='pyephem pytest-mpl wcsaxes astroquery'
 
         # Try all python versions with the latest numpy
         # TODO: add the `--open-files` option back once this issue has been resolved:
         # https://github.com/astropy/astroplan/pull/83#issuecomment-136129489
         - os: linux
           env: PYTHON_VERSION=2.7 SETUP_CMD='test -V'
+               CONDA_DEPENDENCIES='pytz'
+               PIP_DEPENDENCIES=''
+
+        # There isn't numpy 1.10 and python 3.3 combination in conda or
+        # astropy-ci-extras, yet. Update this when it becomes available
```

Comment:
Please remove the "update this when it becomes available".

There's very few Python 3 users on Python 3.3:
http://astrofrog.github.io/blog/2015/05/09/2015-survey-results/
It's OK to keep 3.3 in the build matrix as long as it doesn't generate extra work to support, but we don't have to add more Python 3.3 builds in the future.